src/serlio/materials/MaterialUtils.h

```diff
@@ -32,4 +32,6 @@ std::wstring synchronouslyCreateShadingEngine(const std::wstring& desiredShading
 std::filesystem::path getStingrayShaderPath();
 
 bool textureHasAlphaChannel(std::wstring path);
```

Comment:
should use the opportunity to make this argument a const ref

Reply:
done